statistics/feedback.go

```diff
@@ -616,7 +588,7 @@ func encodeIndexFeedback(q *QueryFeedback) *queryFeedback {
 	pb := &queryFeedback{}
 	var pointCounts []int64
 	for _, fb := range q.feedback {
-		if bytes.Equal(kv.Key(fb.lower.GetBytes()).PrefixNext(), fb.upper.GetBytes()) {
+		if bytes.Compare(kv.Key(fb.lower.GetBytes()).PrefixNext(), fb.upper.GetBytes()) >= 0 {
```

Comment:
Will there be `> 0` cases?

Reply:
Yes, when we set the point queries at [here](https://github.com/pingcap/tidb/pull/7488/files#diff-c3358fcebf9bb42d1dfce3c5ce8ec92aR977), the lower is equal to upper, so after prefix next, it will be `>0`.

Reply:
So the `lower` and `upper` is both inclusive?

Reply:
The lower is inclusive, the upper can be inclusive or exclusive.